test/util.go

```diff
@@ -128,6 +130,23 @@ func MoveMouse(c fyne.Canvas, pos fyne.Position) {
 	}
 }
 
+// NewCheckedImage returns a new black/white checked image with the specified size
+// and the specified amount of horizontal and vertical tiles.
+func NewCheckedImage(w, h, hTiles, vTiles int) image.Image {
```

Comment:
I'm not sure this fits with the rest of the test utilities that help people to test their apps.
This is more like something that we need to test the framework, so does it belong internal instead?